metplotpy/contributed/weather_regime/plot_weather_regime.py

```diff
@@ -0,0 +1,99 @@
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+from pylab import *
+import cmocean
```

Comment:
need to add cmocean to METplotpy and METcalcpy documentation for required, installed packages.

Reply:
I added cmocean to the Users Guide for METplotpy and METcalcpy in the develop branch